oci/layout/oci_dest_test.go

```diff
@@ -108,6 +109,27 @@ func TestPutManifestTwice(t *testing.T) {
 	assert.Equal(t, 2, len(index.Manifests), "Unexpected number of manifests")
 }
 
+func TestPutTwoDifferentTags(t *testing.T) {
+	ref, tmpDir := refToTempOCI(t)
+	defer os.RemoveAll(tmpDir)
+
+	ociRef, ok := ref.(ociReference)
+	require.True(t, ok)
+
+	putTestConfig(t, ociRef, tmpDir)
+	putTestManifest(t, ociRef, tmpDir)
+
+	// add the same manifest with a different tag; it shouldn't get overwritten
+	ociRef.image = "zomg"
```

Comment:
References are conceptually immutable. This should work similarly, and be more representative, with a
```go
   ref2, … = NewReference(tmpDir, "zomg")
```

Reply:
Done.